corehq/ex-submodules/casexml/apps/phone/tests/data/case_relationship_tests.json

```diff
@@ -849,5 +849,71 @@
         "outcome": [
             "a", "b"
         ]
+    },
+    {
+        "name": "test_reach_child_to_mother_parent",
+        "owned": [
+            "claim"
+        ],
+        "closed": [
+            "p_mother"
+        ],
+        "subcases": [
+            ["p_child", "p_mother"]
+        ],
+        "extensions": [
+            ["claim", "house"],
+            ["p_mother", "house"],
+            ["p_child", "house"],
+            ["child_health", "p_child"],
+            ["measurement", "child_health"]
+        ],
+        "outcome": [
+            "claim", "house", "p_mother", "p_child", "child_health", "measurement"
+        ]
+    },
+    {
+        "name": "test_reach_child_to_mother_extension",
+        "owned": [
+            "claim"
+        ],
+        "closed": [
+            "p_mother"
+        ],
+        "subcases": [
+        ],
+        "extensions": [
+            ["claim", "house"],
+            ["p_mother", "house"],
+            ["p_child", "house"],
+            ["p_child", "p_mother"],
+            ["child_health", "p_child"],
+            ["measurement", "child_health"]
+        ],
+        "outcome": [
+            "claim", "house", "p_mother", "p_child", "child_health", "measurement"
```

Comment:
In this test, `p_mother` is still synced because `p_child` is open and open extension cases always pull in their super